### ISSUE TITLE
Feature request: Drawbridge support

### ISSUE THREAD

**Issue by retrorepair** [libretro/libretro-uae#642]
Drawbridge is built into WinUAE and is in the libretro source too but I don't see any options to configure it.

I would add support myself but not sure where I'd start.

For those unaware, this is what it does:

https://youtu.be/2E-6gG1Rwsw

**Comment by sonninnos** [comment]
Yep, `FLOPPYBRIDGE` is not enabled here. Just need to enable the `sysconfig.h` flag and fix any possible build fails.

Some rarely needed obscure stuff are disabled on purpose to keep the core size down, and some are disabled simply because build fails require too much work.

**Comment by retrorepair** [comment]
Ah ok, makes perfect sense. I'll see how I get on then, been a long time since I've compiled a core!

**Comment by retrorepair** [comment]
> Yep, `FLOPPYBRIDGE` is not enabled here. Just need to enable the `sysconfig.h` flag and fix any possible build fails.
> 
> Some rarely needed obscure stuff are disabled on purpose to keep the core size down, and some are disabled simply because build fails require too much work.

Ok, tried this and it instantly fails saying it can't find the include <functional> which I understand is a C++ header, not sure what to install or change without completely breaking things (I'm very dubious about installing anything on linux!). Would you have any ideas?

**Comment by retrorepair** [comment]
If I skip that include, it comolains about vector and string includes so thinking there's something missing here

**Comment by sonninnos** [comment]
You are on your own there. After looking at the video a bit I noticed that even WinUAE requires a plugin, and by the look of the plugin the feature is well out of scope for libretro, since the Disc Control interface does not allow anything so complicated, which means all usage would be via uae configuration file only.

Feel free to find solutions of course.

**Comment by retrorepair** [comment]
Yeah I think your right. In all honesty I think starting from scratch would be easier. I'll see what comes of some experimenting. I had partial success with my own interface some years ago, I may resurrect that instead.

**Comment by retrorepair** [comment]
I've discovered Amiberry has floppybridge support too and requires no external plugin. I may see how it's implimented and integrate that into p-uae instead.

Hopefully it won't be too painful and could simply require an option for drawbridge, greaseweazle or supercard pro (think that's what it's called). It's a much simpler implementation on Amiberry.
